conf/test_full.config

```diff
@@ -18,7 +18,9 @@ params {
     input        = 'https://raw.githubusercontent.com/nf-core/test-datasets/scrnaseq/samplesheet_2.0_full.csv'
 
     // Genome references
-    genome       = 'GRCh38'
+    // genome       = 'GRCh38' // TODO: Fix: incompatible with new star version
+    fasta           = 'https://ftp.ensembl.org/pub/release-110/fasta/homo_sapiens/dna/Homo_sapiens.GRCh38.dna.primary_assembly.fa.gz'
+    gtf             = 'https://ftp.ensembl.org/pub/release-110/gtf/homo_sapiens/Homo_sapiens.GRCh38.110.gtf.gz'
```

Comment:
I don't think we should 'fix' this by overriding the test dataset. 
Please either update the STAR module such that it uses a version compatible with iGenomes, or alternatively still use iGenomes, but only retrieve fasta/gtf from there, but not the STAR index. 

That way it should work for all users.

Reply:
did the latter: https://github.com/nf-core/scrnaseq/pull/369/commits/84a83c29cac1f0f62191945fce7e41c6bbf74e41